API/api/repositories/auth_repository.py

```
@@ -0,0 +1,15 @@
+# api/repositories/auth_repository.py
+
+class Auth_Repository:
+    def __init__(self):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)